weblate/auth/models.py

```diff
@@ -610,10 +610,10 @@ def get_author_name(self, email=True):
 
 class AutoGroup(models.Model):
     match = RegexField(
-        verbose_name=_("E-mail regular expression"),
+        verbose_name=_("Regular expression for e-mail address"),
```

Comment:
This now nearly matches `help_text`. In case we cannot have a shorter label here the help text is not really needed and can be dropped.